packages/portfolio-contract/src/portfolio.exo.ts

```diff
@@ -464,6 +507,18 @@ export const preparePortfolioKit = (
         getTargetAllocation() {
           return this.state.targetAllocation;
         },
+        incrementNonceForChain(chainName: AxelarChain) {
+          const { axelarChainNonces } = this.state;
+          const currentNonce = axelarChainNonces.has(chainName)
+            ? axelarChainNonces.get(chainName) + 1
+            : 1;
```

Comment:
calling this `currentNonce` seems odd to me. I'd expect to see `.set(chain, currentNonce + 1)`

 maybe `noncePost`?